discovery-provider/alembic/trigger_sql/handle_repost.sql

```diff
@@ -117,16 +117,17 @@ begin
 	            r.repost_item_id = new.repost_item_id
 	            and new.created_at - INTERVAL '1 month' < r.created_at
 	            and new.created_at > r.created_at
-				and r.is_delete is false
-				and r.is_current is true
+              and r.is_delete is false
+              and r.is_current is true
+              and r.repost_type = new.repost_type
 	            and r.user_id in (
 	                select
 	                    followee_user_id
 	                from follows
 	                where
 	                    follower_user_id = new.user_id
-						and is_delete is false
-						and is_current is true
+                      and is_delete is false
```

Comment:
i dont understand why git formats this so weirdly.. i swear it looks right on my ide 
<img width="500" alt="Screenshot 2023-02-22 at 2 23 08 PM" src="https://user-images.githubusercontent.com/109105561/220736931-1bd7fb53-5495-4952-89b8-a13144ec23be.png">